trulens_eval/trulens_eval/schema.py

```diff
@@ -589,6 +611,13 @@ class FeedbackDefinition(pyschema.WithClassInfo, serial.SerialModel, Hashable):
     feedback_definition_id: FeedbackDefinitionID
     """Id, if not given, uniquely determined from content."""
 
+    if_exists: Optional[serial.Lens] = None
```

Comment:
does this produce any warning/error if a needed Lens doesn't exist? If so, what is that message? Can we use it to point people to find the correct lens either in UI or with ipytree widget?

Reply:
You mean if the thing pointed to by `if_exists` does not exist in the record? If so, yes:

```python
logger.warning(f"Feedback %s skipped as %s does not exist.", self.name, self.if_exists)
```

Reply:
Unsure the best thing to point people to. Have mentioned the green texts in dashboard before but need better solutions.